package.json

```diff
@@ -3,7 +3,10 @@
   "license": "GPL-3.0-or-later",
   "main": "src/index.js",
   "devDependencies": {
-    "@wordpress/scripts": "^26.14.0",
+    "@babel/preset-env": "^7.26.7",
+    "@babel/preset-react": "^7.26.3",
+    "@wordpress/scripts": "^30.10.0",
```

Comment:
Please bear in mind that current script v30.10.0 is compatible with WordPress 6.7 and higher